{"name": "Параметризованная проверка невозможности авторизации при передаче неверного логина или пароля", "status": "passed", "description": "Создаем курьера, пытаемся авторизоваться передавая неверный логин или пароль,                         проверяем что код ответа: 404, текст ответа: \"Учетная запись не найдена\"", "parameters": [{"name": "incorrect_argument", "value": "'login'"}], "start": 1710001660593, "stop": 1710001680270, "uuid": "db392580-0fba-46ee-93dd-ab5e255078bf", "historyId": "506de2072404a1264b533f1dfbcca130", "testCaseId": "494d196ed4678e26fab0bc9442c7c320", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_courier_with_incorrect_login_or_password_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-1VNC7DB"}, {"name": "thread", "value": "34604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}